### ISSUE TITLE
":" in pair name

### ISSUE THREAD

**Issue by Anarchotrader** [jnidzwetzki/bitfinex-v2-wss-api-java#209]
The point is the length of the array, which is incorrect because of the colon.

For example, an attempt to use such a pair causes errors "BTCF0:USTF0".

Error in method **BitfinexCandlestickSymbol.fromBitfinexString(String symbol)**

log

```
21:25:36.697 [ tyrus-jdk-client-3] ERROR com.github.jnidzwetzki.bitfinex.v2.SimpleBitfinexApiBroker - OnError called java.lang.IllegalArgumentException: Unable to parse: trade:1m:tBTCF0:USTF0
at com.github.jnidzwetzki.bitfinex.v2.symbol.BitfinexCandlestickSymbol.fromBitfinexString(BitfinexCandlestickSymbol.java:70)
at com.github.jnidzwetzki.bitfinex.v2.callback.command.SubscribedCallback.handleCandlesCallback(SubscribedCallback.java:89)
at com.github.jnidzwetzki.bitfinex.v2.callback.command.SubscribedCallback.handleChannelData(SubscribedCallback.java:56)
at com.github.jnidzwetzki.bitfinex.v2.SimpleBitfinexApiBroker.handleCommandCallback(SimpleBitfinexApiBroker.java:552)
at com.github.jnidzwetzki.bitfinex.v2.SimpleBitfinexApiBroker.websocketCallback(SimpleBitfinexApiBroker.java:530)
at com.github.jnidzwetzki.bitfinex.v2.WebsocketClientEndpoint.onMessage(WebsocketClientEndpoint.java:117)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.glassfish.tyrus.core.AnnotatedEndpoint.callMethod(AnnotatedEndpoint.java:552)
at org.glassfish.tyrus.core.AnnotatedEndpoint.access$100(AnnotatedEndpoint.java:65)
at org.glassfish.tyrus.core.AnnotatedEndpoint$WholeHandler$1.onMessage(AnnotatedEndpoint.java:651)
at org.glassfish.tyrus.core.TyrusSession.notifyMessageHandlers(TyrusSession.java:559)
at org.glassfish.tyrus.core.TyrusEndpointWrapper.onMessage(TyrusEndpointWrapper.java:855)
at org.glassfish.tyrus.core.TyrusWebSocket.onMessage(TyrusWebSocket.java:196)
at org.glassfish.tyrus.core.frame.TextFrame.respond(TextFrame.java:115)
at org.glassfish.tyrus.core.ProtocolHandler.process(ProtocolHandler.java:819)
at org.glassfish.tyrus.client.TyrusClientEngine$TyrusReadHandler.handle(TyrusClientEngine.java:724)
at org.glassfish.tyrus.container.jdk.client.ClientFilter.processRead(ClientFilter.java:204)
at org.glassfish.tyrus.container.jdk.client.Filter.onRead(Filter.java:111)
at org.glassfish.tyrus.container.jdk.client.Filter.onRead(Filter.java:113)
at org.glassfish.tyrus.container.jdk.client.SslFilter.handleRead(SslFilter.java:383)
at org.glassfish.tyrus.container.jdk.client.SslFilter.processRead(SslFilter.java:345)
at org.glassfish.tyrus.container.jdk.client.Filter.onRead(Filter.java:111)
at org.glassfish.tyrus.container.jdk.client.Filter.onRead(Filter.java:113)
at org.glassfish.tyrus.container.jdk.client.TransportFilter$4.completed(TransportFilter.java:294)
at org.glassfish.tyrus.container.jdk.client.TransportFilter$4.completed(TransportFilter.java:278)
at java.base/sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:127)
at java.base/sun.nio.ch.Invoker$2.run(Invoker.java:219)
at java.base/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)



21:25:36.712 [ tyrus-jdk-client-3] ERROR com.github.jnidzwetzki.bitfinex.v2.SimpleBitfinexApiBroker - Unable to determine symbol for channel 76046 / data is [76046,[[1611771900000,30243,30220,30281,30220,13.79159809],
```

**Comment by jnidzwetzki** [comment]
@Anarchotrader I have merged a PR addressing this issue (https://github.com/jnidzwetzki/bitfinex-v2-wss-api-java/pull/211) a few minutes ago. Can you try to reproduce this issue with the current development version?

**Comment by Anarchotrader** [comment]
> @Anarchotrader I have merged a PR addressing this issue (#211) a few minutes ago. Can you try to reproduce this issue with the current development version?

On this version everything works!
